zerver/tests/test_hooks.py

```diff
@@ -14,10 +15,10 @@ class WebhookTestCase(AuthedTestCase):
     If you create your url in uncommon way you can override build_webhook_url method
     In case that you need modify body or create it without using fixture you can also override get_body method
     """
-    STREAM_NAME = None
+    STREAM_NAME = None # type: text_type
```

Comment:
It's hard to know, since mypy doesn't check Optional yet.  I think probably these should be `Optional[text_type]` technically, but I wouldn't be surprised if mypy will know to infer the `Optional` for variables initialized to None.  I went ahead and changed these.